community/kernel/src/main/java/org/neo4j/kernel/internal/Version.java

```diff
@@ -170,6 +149,20 @@ public static void main( String[] args )
     }
 
     static final String KERNEL_ARTIFACT_ID = "neo4j-kernel";
-    private static final Version KERNEL_VERSION = new Version( KERNEL_ARTIFACT_ID,
-            Version.class.getPackage().getImplementationVersion() );
+    private static final Version KERNEL_VERSION = new Version( KERNEL_ARTIFACT_ID, getMavenVersion() );
```

Comment:
hm...for example:
```
2016-09-19 20:23:54.589+0000 INFO  [o.n.k.i.DiagnosticsManager] Kernel version: 3.0.6-ENG01,5515f38aa3fbea663116cb3349f01d67c7242cca
```
this is actual line from log file where you can see kernel version together with commit printed out - and it's there; also it's sometimes handy to see commit id since some customers from time to time have special engineering releases